gnd/src/main/java/com/google/android/gnd/ui/editobservation/EditObservationViewModel.java

```diff
@@ -169,7 +171,11 @@ void initialize(EditObservationFragmentArgs args) {
   }
 
   Optional<Response> getResponse(String fieldId) {
-    return Optional.ofNullable(responses.get(fieldId));
+    if (responses.isEmpty()) {
```

Comment:
Please add comments to clarify when one case would occur vs the other?

Reply:
Done